src/applications/personalization/profile/tests/e2e/address-validation/low-confidence.cypress.spec.js

```diff
@@ -0,0 +1,48 @@
+import { setUp } from 'applications/personalization/profile/tests/e2e/address-validation/setup';
+
+describe('Personal and contact information', () => {
+  describe('when entering a low confidence address that needs confirmation', () => {
+    it('should successfully update on Desktop', () => {
```

Comment:
and this to something like `should show the validation view and successfully update`?